internal/controller/backup/collect_test.go

```diff
@@ -125,27 +125,18 @@ func Test_sortDedup(t *testing.T) {
 				{
 					MatchLabels: map[string]string{"cluster.x-k8s.io/provider": "control-plane-k0smotron"},
 				},
-				{
-					MatchLabels: map[string]string{"cluster.x-k8s.io/provider": "infrastructure-azure"},
-				},
```

Comment:
what is the reason for removing the unrelated data from the test? besides, this will be in the final spec, so please, revert the changes

Reply:
idea is to have no mention of other that `aws` providers in tests, it does not affect anything but just keeps codebase consistent and makes us remeber that we are provider-agnostic

Reply:
1. it is the test data (one could even name it `golden`)
2. the actual backup has these label selectors (a `clusterdeployment` object always has a provider)
3. please keep them since the data has no relation to providers or to plugins

Reply:
if you want to rename `azure` to `aws` please go ahead, but the labels and the value-style should persist

Reply:
I think there are aws values in there, sure I can rename, no prob